### ISSUE TITLE
Type hints are not readily available through Pylance in VS Code

### ISSUE THREAD

**Issue by RA80533** [aio-libs/aioodbc#421]
Alternate title: "aioodbc does not support PEP 561"

[PEP 561 – Distributing and Packaging Type Information] defines the mechanisms for distributing packages with type hints:

> Package maintainers who wish to support type checking of their code MUST add a marker file named `py.typed` to their package supporting typing. This marker applies recursively: if a top-level package includes it, all its sub-packages MUST support type checking as well.

Since aioodbc does not contain a `py.typed` marker file, developers may find themselves without any type hints for the package. Developers are unable to take advantage of the type annotations already part of the project.

Most typed packages on PyPI publish place their `py.typed` file in the namespace directory (e.g., "\<root\>/aioodbc") to better centralize the type checking to the package source code. It would be a huge improvement to the developer experience if the aioodbc project included a `py.typed` marker file alongside the package.

[PEP 561 – Distributing and Packaging Type Information]:
https://peps.python.org/pep-0561/

**Comment by jettify** [comment]
Yeah, `py.typed` missing because not everything typed, I was slowly moving that direction. Happy to accept any PRs.
